CONTRIBUTORS.md

```diff
@@ -5,8 +5,9 @@
 ## Signed-off-by
 
 - Copyright 2023 Resource Management Associates Inc.
```

Comment:
@adamkorynta  FYI, technical you're a maintainer of the project as an individual. Not that this is an inappropriate line but though I'd mention it if you wanted make that more clear.

Reply:
I'm not sure I understand what you're asking me to put instead. All the work done so far is under contract through RMA. I followed the same format as on HEC's other repositories (and OpenDCS) doing work under contract.

Reply:
Nothing specific. Mostly just letting you know if you wanted to list yourself separate from RMA but certainly not a requirement.